Page not found

miaerbus opened the pull request:
Partially fixes #44

@stefanb what exactly did you mean by "offer the user a link to search page with preselected doctor type". Did you mean a link to the homepage with preselected doctor type filter (maybe as a query param) or a additional search field in the 404 page or something else?

@bananica maybe a different image for 404 page, what do you think?

stefanb commented:
Just a plain 404 page in needed, but should be rarely seen by users

stefanb commented on src/components/SEO/index.js:
```diff
@@ -0,0 +1,16 @@
+import { Helmet } from 'react-helmet-async';
+
+export const Static = function Static({ children }) {
+  return (
+    <Helmet>
+      <html lang="en" />
+      <title>Hello from React Helmet</title>
+      <meta name="description" content="Basic example" />
+      {children}
+    </Helmet>
+  );
+};
```
Do we need this with such values?

jalezi replied:
No. Sorry, we don't need Static component at the moment. I wasn't sure what I will use and forgot to delete it.

stefanb replied:
```suggestion
```

stefanb commented:
Some translations are missing:
![image](https://user-images.githubusercontent.com/319826/144876963-bd0b06e1-8ac0-47cb-b6e8-92ded863d408.png)

Please check the JSON structure (`pageNotFound` vs `SEO.title` groups)

`/sl/eNotFound` comes from language switching, "/sl/" overwriting start of "/PageNotFound" URL.

miaerbus commented:
Possible fix for the problem https://github.com/sledilnik/zdravniki/commit/2068d2d56036be5edd6f1961f2578be8effcd0ad, can I merge it to this branch?

stefanb commented:
@miaerbus sure, whatever is needed. A separatee `404` branch on it's own makes no sense.

bananica commented:
@miaerbus I'll try to come up with a new image/illustration for 404, I love interesting 404 pages, 404 doctor not found ;)

miaerbus commented:
Added fix for #58

miaerbus commented:
- Fixed an issue that always redirected to 404 on doctor page.
- Switched to `useDoctors` hook because `useFilter` was returning only GP doctors, dentists and gynaecologists were missing.
- I think 'react-router-dom' package is enough, maybe we can remove 'react-router'
